File: Linear_Search.py
def linear_search(arr,find_element):
    for i in range(len(arr)):
        if arr[i]==find_element:
            return i
    return -1
   
arr=[1,2,3,4,5,6,7,8,10]
find_element=5
print(f"Original List:{arr}")
result=linear_search(arr,find_element)
if(result==-1):
    print("Element not found")
else:
    print("Element found at index:",result)    

File: Linear_Search.cpp
#include<bits/stdc++.h>
using namespace std;

int main(){
    vector<int>arr={5,7,2,3,5,1,4,8};
    int n=arr.size();
    int target=3;
    for(int i=0;i<n;i++){
        if(arr[i]==target){
            printf("Element found at index %d\n",i);
            return 0;
        }
    }
    printf("Element not found\n");
    return 0;
}
